### PR TITLE
Fixed download by working around environment issues

### PR DESCRIPTION
**Pull request by jbeez**
Had to work around plex python environment not reading my variables, used ln -s to move python27.zip file to python binary's current directory, seems to work great no issues in testing.

### CODE REVIEW

**Comment by jarodwsams** [comment]
I can confirm that this commit fixes #57.

**Comment by Zzyzx-Wolfe** [comment]
Tested and working for me. Though, as a small note, if someone originally installed the non-PlexPass package originally, it won't find their token.
`/usr/local/plexdata/Plex Media Server/Preferences.xml` is a third possible path, for completeness. They should pass `-n` if they want the non-PlexPass version.

#54 was my hacky fix of that issue a while back.

File: PMS_Updater.sh [hunk 1]
```diff
@@ -6,7 +6,13 @@ VERBOSE=1
 REMOVE=1
 LOGGING=1
 
-PLEXTOKEN="$(sed -n 's/.*PlexOnlineToken="//p' /Plex\ Media\ Server/Preferences.xml | sed 's/\".*//')"
+#check if we're in pms plugin, or standard jail
+if [ -e "/Plex Media Server/Preferences.xml" ]; then {
+        PREFS="/Plex Media Server/Preferences.xml"
+} else {
+        PREFS="/usr/local/plexdata-plexpass/Plex Media Server/Preferences.xml"
```
Comment (mweber03) [inline PMS_Updater.sh:13]: In order to support non-plexpass (`-n`) can you swap `plexdata-plexpass` to `$PMSDATAFOLDER` which is set in the if block on 207

```sh
PMSDATAFOLDER="plexdata-plexpass"
...
PMSDATAFOLDER="plexdata"
```

**Comment by CognitiveFeline** [comment]
works great

**Comment by MAGICCC** [comment]
@jbeez would you like to implent the check from @Zzyzx-Wolfe so we can merge this or would it better to use #54?

**Comment by Zzyzx-Wolfe** [comment]
> @jbeez would you like to implent the check from @Zzyzx-Wolfe so we can merge this or would it better to use #54?

It would probably be better if we used the `$PMSDATAFOLDER` variable. If I have time, I can add that in to this.

**Comment by MAGICCC** [comment]
> 
> 
> > @jbeez would you like to implent the check from @Zzyzx-Wolfe so we can merge this or would it better to use #54?
> 
> Looking at @jbeez's fork, it looks like they added it, but have not added the commits to the PR: https://github.com/jbeez/PMS_Updater/tree/TokenAuth

Ah yes, you are right :(

**Comment by jbeez** [comment]
Hey guys, I'll try to review this shortly when I get a little time and add the changes to this PR :)

File: PMS_Updater.sh (hunk 1 above)
Comment (jbeez) [reply PMS_Updater.sh:13]: @mweber03 I don't see PMSDATAFOLDER anywhere, I do see $PMSPARENTPATH/$PMSLIVEFOLDER ?  I could perhaps leverage that.  I might not be looking at the same version of the file though.

**Review by Zzyzx-Wolfe — APPROVED** [review]
Thanks for adding this. Fixes my only concern.

**Comment by MAGICCC** [comment]
Thanks!

**Pull request merged**